src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3257,7 +3267,6 @@ public partial struct UInt64 : System.IComparable, System.IComparable<ulong>, Sy
     [System.CLSCompliantAttribute(false)]
     public partial struct UIntPtr : System.IEquatable<System.UIntPtr>, System.Runtime.Serialization.ISerializable
     {
-        private int _dummy;
```

Comment:
I am not sure why this got removed. Should it have been?

UIntPtr contains a void* private field:
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/UIntPtr.cs#L23

Reply:
That is incorrect, it should still contain the dummy field here and in IntPtr. See https://github.com/dotnet/buildtools/pull/2003/files#r180160381 where I think you need to fix to address.